sw/airborne/modules/radio_control/rc_datalink.c

```diff
@@ -62,6 +62,29 @@ void rc_datalink_parse_RC_4CH(uint8_t *buf)
       DL_RC_4CH_yaw(buf));
 }
 
+void rc_datalink_parse_RC_6CH(uint8_t *buf)
+{
+#ifdef RADIO_CONTROL_DATALINK_LED
+  LED_TOGGLE(RADIO_CONTROL_DATALINK_LED);
+#endif
+  parse_rc_6ch_datalink(DL_RC_6CH_mode(buf),
+      DL_RC_6CH_throttle(buf),
+      DL_RC_6CH_roll(buf),
+      DL_RC_6CH_pitch(buf),
+      DL_RC_6CH_yaw(buf),
+      DL_RC_6CH_kill(buf),
+      DL_RC_6CH_aux7(buf));
+}
```

Comment:
can we remove this one if the generic version is working ?

Reply:
Yes!